scripts/storage-emulator-integration/tests.ts

```diff
@@ -1,2228 +0,0 @@
-              .ref(filename)
-              .updateMetadata({
-                contentType: "text/plain",
-                customMetadata: {
-                  removeMe: null as any,
-                },
-              });
-          }, filename);
-
-          expect(nulledMetadata.customMetadata.removeMe).to.equal(undefined);
-        });
-
-        it("throws on non-existent file", async () => {
-          await signInToFirebaseAuth(page);
-          const err = await page.evaluate(async () => {
-            try {
-              return await firebase
-                .storage()
-                .ref("testing/thisFileDoesntExist")
-                .updateMetadata({
-                  contentType: "application/awesome-stream",
-                  customMetadata: {
-                    testable: "true",
-                  },
-                });
-            } catch (_err) {
-              return _err;
-            }
-          });
-
-          expect(err).to.not.be.empty;
-        });
-      });
-
-      describe("deleteFile", () => {
-        it("should delete file", async () => {
-          await signInToFirebaseAuth(page);
-          await page.evaluate((filename) => {
-            return firebase.storage().ref(filename).delete();
-          }, filename);
-
-          const error = await page.evaluate((filename) => {
-            return new Promise((resolve) => {
-              firebase
-                .storage()
-                .ref(filename)
-                .getDownloadURL()
-                .catch((err) => {
-                  resolve(err.message);
-                });
-            });
-          }, filename);
-
-          expect(error).to.contain("does not exist.");
-        });
-
-        it("should not delete file when security rule on resource object disallows it", async () => {
-          await uploadText(page, "delete/disallowIfContentTypeText", "some-content", undefined, {
-            contentType: "text/plain",
-          });
-
-          const error: string = await page.evaluate(async (filename) => {
-            try {
-              await firebase.storage().ref(filename).delete();
-              return "success";
-            } catch (err) {
-              if (err instanceof Error) {
-                return err.message;
-              }
-              throw err;
-            }
-          }, "delete/disallowIfContentTypeText");
-
-          expect(error).to.contain("does not have permission to access");
-        });
-      });
-    });
-  });
-
-  describe("Non-SDK Endpoints", () => {
-    const filename = "testing/storage_ref/image.png";
-    const encodedFilename = "testing%2Fstorage_ref%2Fimage.png";
-    const imageFilename = writeToFile(
-      "image_base64",
-      Buffer.from(IMAGE_FILE_BASE64, "base64"),
-      tmpDir
-    );
-
-    beforeEach(async () => {
-      await resetEmulatorState();
-      await testBucket.upload(imageFilename, { destination: filename });
-    });
-
-    describe("tokens", () => {
```

Comment:
I think this tokens section might have been missed in the migration, I can't seem to find these tests anymore

Reply:
good catch!